integration/spark/src/test/scala/org/apache/carbondata/spark/util/BadRecordUtil.scala

```diff
@@ -68,4 +71,32 @@ object BadRecordUtil {
     badRecordLocation = badRecordLocation + "/" + dbName + "/" + tableName
     FileFactory.deleteAllCarbonFilesOfDir(FileFactory.getCarbonFile(badRecordLocation))
   }
+
+  def createCSV(rows: ListBuffer[Array[String]], csvPath: String): Unit = {
+    val out = new BufferedWriter(new FileWriter(csvPath))
```

Comment:
correct the formatting of this method and may be you can use try-with-resource here, you can check once

Reply:
could not find proper try-with-resource in scala. keeping it the same.